### ISSUE TITLE
It seems that only works with Python kernel, would you like to make it works with more kernels, such as IJulia?

### ISSUE THREAD

**Issue by zombie110year** [mwakaba2/jupyterlab-notifications#6]
I installed this plugin by `pip install`, version `0.1.4`, jupyter lab version `3.0.14`.

There were no kernel-specified code, but I don't know why it doesn't work with Julia kernel.

Cell Code:

```julia
sleep(15)
```

Cell Metadata

```json
{
    "execution": {
        "iopub.status.busy": "2021-04-15T02:16:43.514000+08:00",
        "iopub.execute_input": "2021-04-15T02:16:44.663000+08:00",
        "iopub.status.idle": "2021-04-15T02:17:00.266000+08:00",
        "shell.execute_reply": "2021-04-15T02:17:00.120000+08:00"
    },
    "tags": []
}
```

Notebook Metadata

```json
{
    "kernelspec": {
        "name": "julia-1.6",
        "display_name": "Julia 1.6.0",
        "language": "julia"
    },
    "language_info": {
        "file_extension": ".jl",
        "name": "julia",
        "mimetype": "application/julia",
        "version": "1.6.0"
    }
}
```

But with Python3 kernel, it works.

All Firefox and Microsoft Edge have this problem on Windows 10.

**Comment by zombie110year** [comment]
I learned to sign certifications and configure nginx with 3 hours, but meet this problem, so sad. 😢

**Comment by mwakaba2** [comment]
@zombie110year thanks for reporting this! I'll try to reproduce this in a Julia Kernel.

**Comment by mwakaba2** [comment]
@zombie110year I released a new version with the fix (v0.1.5). Please use the following command to upgrade the lab extension.

$ pip install --upgrade jupyterlab-notifications

In short, the issue was the extension wasn't able to calculate the cell execution time because the Julia Kernel cell execution metadata didn't have the `shell.execute_reply.started` attribute.
